README.md

```diff
@@ -4,48 +4,57 @@ The purpose of this repo is to build, test and run omnibuses.
 
 ## Omnibus
 
-The main purpose of omnibus is to prepare the EVM script that will be executed if the vote is successful.
-During the run, omnibus will call newVote function of the Voting contract with the prepared script and description.
-Voting EVM script will be built from items that are defined in the omnibus.
+The main purpose of the omnibus is to prepare the EVM script that will be executed if the vote is successful. The voting EVM script is built from the omnibus items defined in the omnibus. During the run, the omnibus will call the
+```
+newVote(bytes executionScript, string metadata, bool ???, bool ???)
+```
+function of the voting contract with the prepared script and description.
+
+## Omnibus Item
+
+Each omnibus is made up of items. An omnibus item is the basic building block of each omnibus. It represents a single on-chain action (such as changing protocol settings, granting or revoking access, transferring tokens, etc). In code, it is represented as an object containing
+* `title` - arbitrary name of the current item
+* `evmCall` - ????
```

Comment:
@Ivan-Feofanov please help with the `evmCall` field desc here

Reply:
`evmCall` - it's an EVM call script of specific item that will be added to the entire voting script and executed after voting enact. Call encoding funtion can be found [here](https://github.com/lidofinance/depot/blob/811b1df686e935e2df71c1ed5168271afc6e6874/src/votes/vote-script.ts#L105)